internal/session/repository_session.go

```diff
@@ -64,12 +64,14 @@ func (r *Repository) CreateSession(ctx context.Context, sessionWrapper wrapping.
 		return nil, nil, errors.New(ctx, errors.InvalidParameter, op, "missing expiration time")
 	}
 
+	opts := getOpts(opt...)
+
 	id, err := newId()
 	if err != nil {
 		return nil, nil, errors.Wrap(ctx, err, op)
 	}
 
-	privKey, certBytes, err := newCert(ctx, sessionWrapper, newSession.UserId, id, newSession.ExpirationTime.Timestamp.AsTime())
+	privKey, certBytes, err := newCert(ctx, sessionWrapper, newSession.UserId, id, opts.withWorkerAddresses, newSession.ExpirationTime.Timestamp.AsTime())
```

Comment:
Is this really optional?  Feels like a required parameter.

Reply:
Yes, it's optional. I just didn't want to introduce an options pattern into `newCert` -- but I can easily do so if you prefer!